datahub-web-react/src/app/settings/AccessTokens.tsx

```diff
@@ -170,16 +172,14 @@ export const AccessTokens = () => {
             }
         }
     }, [canManageToken, owner]);
-
+    
     const renderSearchResult = (entity: any) => {
-        const {
-            editableProperties,
-            info: { displayName },
-        } = entity;
+        const { editableProperties } = entity;
+        const displayNameSearchResult = entityRegistry.getDisplayName(EntityType.CorpUser, entity);
```

Comment:
do we always know this will be a corpuser or could it be a corpGroup? I would just pass in `entity.type` instead of hardcoding `EntityType.CorpUser`

Reply:
Yes we always showing corp users data in dropdown.